app/views/protocols/index.html.erb

```diff
@@ -48,16 +48,28 @@
             </a>
           <% end %>
         </div>
+<div id="import-export-protocols" class="btn-group" role="group">
```

Comment:
Is this indentation ok?

Reply:
fixed, i was testing something and forgot to indent it properly